content/docs/user-guide/gems/reference/aws/aws-core/getting-started.md

```diff
@@ -27,6 +27,22 @@ To get started using AWS Gems with AWS services in your O3DE project, complete t
 
 You should now be able to utilize AWS functions in Lua script, Script Canvas, or C++ to communicate with your AWS resources. See [Scripting with AWS Core](./scripting/) for scripting examples.
 
+## Prevent calls to Amazon EC2 Instance Metadata Service
+
+AWS Gems use the [AWS C++ SDK](https://github.com/aws/aws-sdk-cpp) to call AWS resources. Unless your project is running on Amazon EC2 compute, it's recommended that you turn off the [AWS_EC2_METADATA_DISABLED](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html) environment variable. 
```

Comment:
since `AWS_EC2_METADATA_DISABLED` needs to be set "true" to disable it, feels wrong to phrase it as "turning it off". 

Maybe we should just say "it's recommended that you set the [AWS_EC2_METADATA_DISABLED](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html) environment variable to `true`. Setting this environment variable will prevent..."

Reply:
Fixed below.